code/modules/supermatter/supermatter.dm

```diff
@@ -435,7 +435,8 @@
 	radio_controller.remove_object(src, frequency)
 	frequency = new_frequency
 	if(frequency)
-		radio_connection = radio_controller.add_object(src, frequency, filter = RADIO_ATMOSIA)
+		radio_connection = radio_controller.add_object(src, frequency, RADIO_SUPERMATTER)
```

Comment:
I'm pretty sure using a filter that is not RADIO_ATMOSIA will make the AAC unable to interface with the supermatter. Niche feature but still a feature.

Reply:
What can the AAC do with the supermatter? I've never noticed it on the command list.

Reply:
You can read its power/integrity/something else

Reply:
I've used supermatter data to set my AAC up to kill the emitters / purge the room in the event of critical integrity levels.